<?xml version="1.0"?>
<launch>    
        
<!--     static_transform_publisher x y z qx qy qz qw frame_id child_frame_id  period_in_ms -->
     <node pkg="tf" type="static_transform_publisher" name="tf_base_link_to_torso" args="0 0 0   0 0 0 1 base_link torso 100" /> 
     <node pkg="tf" type="static_transform_publisher" name="tf_head_to_cam" args="0 0 0.1 0 0 0 1 Head camera_rgb_frame 1000" /> 
    
    <node name="kfusion" pkg="kfusion_ros" type="kfusion_node" output="screen">
        <rosparam file="$(find kfusion_ros)/config/kfusion_params.yaml" command="load"/> 
        
        <param name="depth_topic"    type="str"    value="/camera/depth_registered/image_raw" />
        <param name="cam_info_topic" type="str" value="/camera/depth_registered/camera_info" />
        <param name="rgb_topic"      type="str" value="/camera/rgb/image_rect_color" />


        <param name="publish_volume" type="bool" value="true" />
        <param name="publish_points" type="bool" value="false" />
        <param name="publish_points_rate" type="int" value="20" />
    </node>    
</launch>
